src/components/home/FundingRounds.tsx

```diff
@@ -0,0 +1,128 @@
+import { format } from "date-fns"
+import { useMemo } from "react"
+import Image from "next/image"
+import { cn, titlecase } from "@/lib/utils"
+import { FundingRound } from "@/dummyData/mocks"
+import { Button } from "../ui/button"
+import { Badge } from "../common/Badge"
+
+export const FundingRounds = ({
+  className,
+  fundingRounds,
+}: {
+  className?: string
+  fundingRounds: FundingRound[]
+}) => {
+  const { now, upcoming, past } = useMemo(() => {
+    const n: FundingRound[] = []
+    const u: FundingRound[] = []
+    const p: FundingRound[] = []
+
+    fundingRounds.forEach((round) => {
+      switch (round.status) {
+        case "now":
+          n.push(round)
+          break
+        case "upcoming":
+          u.push(round)
+          break
+        case "past":
+          p.push(round)
+          break
+      }
+    })
+
+    return { now: n, upcoming: u, past: p }
+  }, [fundingRounds])
+
+  const renderSection = (
+    status: "now" | "upcoming" | "past",
+    rounds: FundingRound[],
+  ) => {
+    return (
+      <div className="flex flex-col gap-y-4">
+        <div className="h-8">
+          <h3 className="text-lg font-semibold">{titlecase(status)}</h3>
+        </div>
+        {rounds.map((fundingRound) => (
+          <Round key={fundingRound.number} fundingRound={fundingRound} />
+        ))}
+      </div>
+    )
+  }
+
+  return (
+    <div className={cn("flex flex-col gap-y-12 w-full", className)}>
+      {now.length > 0 && renderSection("now", now)}
```

Comment:
i think it's slightlyyyy more performant if you define these as another JSX component rather than rendering them inline but nbd lolol

Reply:
ah good call, will change that when i connect them to live data